_data/simple-icons.json

```
@@ -10830,6 +10830,21 @@
             "source": "https://about.sourcegraph.com/handbook/marketing/brand/brand_guidelines",
             "guidelines": "https://about.sourcegraph.com/handbook/marketing/brand/brand_guidelines"
         },
+        {
+            "title": "SourceHut",
+            "hex": "000000",
+            "source": "https://sourcehut.org/logo/",
+            "guidelines": "https://sourcehut.org/logo/",
+            "license": {
+                "type": "CC0-1.0",
+                "url": "https://sourcehut.org/logo/"
+            },
+            "aliases": {
+                "aka": [
+                    "sr.ht"
+                ]
+            }
```

```suggestion
            }
```

This is unnecessary. `sr.ht` is not a widely used name.